### PR TITLE
[Core] Fix deprecated numpy type

### PR DESCRIPTION
**Pull request by alxtkr77**
https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations

### CODE REVIEW

File: mlrun/data_types/to_pandas.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def toPandas(spark_df):
         if isinstance(field.dataType, IntegralType) and pandas_col.isnull().any():
             dtype[fieldIdx] = np.float64
         if isinstance(field.dataType, BooleanType) and pandas_col.isnull().any():
-            dtype[fieldIdx] = np.object
+            dtype[fieldIdx] = np.object_
```
Comment (gtopper) [inline mlrun/data_types/to_pandas.py:181]: The recommendation is to replace this deprecated type with a plain `object`, not with the internal type.

```suggestion
            dtype[fieldIdx] = object
```

**Review by gtopper — CHANGES_REQUESTED** [review]

File: mlrun/data_types/to_pandas.py (hunk 1 above)
Comment (alxtkr77) [reply mlrun/data_types/to_pandas.py:181]: Same goes for  np.float64. Lets be consistent

Comment (gtopper) [reply mlrun/data_types/to_pandas.py:181]: Please see https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated:
> For float and complex you can use float64 and complex128 if you wish to be more explicit about the precision.

Comment (gtopper) [reply mlrun/data_types/to_pandas.py:181]: But feel free to change it to `float` if you think that makes more sense.

Comment (alxtkr77) [reply mlrun/data_types/to_pandas.py:181]: np.float64 is in may places - I don't want to change it. So I don't agree with you, but I don't feel it is important enough. Changing it as per your suggestion

**Review by gtopper — APPROVED** [review]

**Pull request merged**